### PR TITLE
KOGITO-4452 Setup LTS PR check

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-4452

In order to test against LTS version of Quarkus.

Command will be `Jenkins run LTS[ tests]`

Related PRs:
- https://github.com/kiegroup/kogito-pipelines/pull/119
- https://github.com/kiegroup/kogito-runtimes/pull/1072
- https://github.com/kiegroup/optaplanner/pull/1201
- https://github.com/kiegroup/kogito-apps/pull/655
- https://github.com/kiegroup/kogito-examples/pull/606

### CODE REVIEW

**Review by msmagnanijr — APPROVED** [review]
lgtm

File: Jenkinsfile [hunk 1]
```diff
@@ -96,32 +134,34 @@ void checkoutRepo(String repo, String dirName=repo) {
...
-    if(extraArgs != ""){
-        mvnCmd += " ${extraArgs}"
-    }
-    dir(directory) {
-        maven.runMavenWithSubmarineSettings(mvnCmd, skipTests)
+MavenCommand getMavenCommand(String directory, boolean addQuarkusVersion=true) {
+    mvnCmd = new MavenCommand(this, ['-fae'])
+                .inDirectory(directory)
+    if (addQuarkusVersion && getQuarkusBranch()) {
+        mvnCmd.withProperty('version.io.quarkus', '999-SNAPSHOT')
```
Comment (rsynek) [inline Jenkinsfile:160]: Please confirm my understanding:
- Quarkus will have an LTS branch different than the `master` branch which we already check via the Quarkus ecosystem CI
- this branch keeps the `999-SNAPSHOT` all the time
Is that correct?

Comment (radtriste) [reply Jenkinsfile:160]: - Yes, this was started due to possibility to migrate Kogito master to Quarkus 1.12 but we still wanted to be sure it works for 1.11. The check will remain and could be useful in the future.
- Quarkus has the particularity to have the same version on all branches, aka 999-SNAPSHOT

Note that the `Quarkus ecosystem CI` does only check kogito runtimes repository. For the whole Kogito ecosystem to be tested, we have nightly run in Jenkins.
This PR check added is just a precheck if needed against Quarkus LTS, if different from the one used in Kogito runtimes.

Comment (radtriste) [reply Jenkinsfile:160]: Note that I am going to update as well the PR template to add the `run LTS check` command

**Comment by rsynek** [comment]
Jenkins run LTS

**Comment by radtriste** [comment]
@rsynek the LTS PR job does not exist yet. will create it manually for now and will be rewritten once we merge this PR ;)

**Comment by radtriste** [comment]
Jenkins run LTS

**Comment by radtriste** [comment]
Jenkins run LTS

**Comment by rsynek** [comment]
> @rsynek the LTS PR job does not exist yet. will create it manually for now and will be rewritten once we merge this PR ;)

Ok, I noticed that on some of the linked PRs the build has been already triggered, so I was curious :)

**Review by rsynek — APPROVED** [review]

**Pull request merged**